glade/adrv9002.glade

```diff
@@ -0,0 +1,2723 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generated with glade 3.22.2 -->
```

Comment:
Why hasn't this been created with Glade 3.8?

Reply:
It's just what you can get with ubuntu 20.04. I even tried to used docker to use older glade for gtk2 but it was not working that good at all. So,  It was easier just using this and run the fixup scripts...

Reply:
If Glade for GTK2 is unreliable/unusable then we should talk about this and decide whether continue using it (and telling people to use it) or not.

Did you also run this fixup script? (https://github.com/analogdevicesinc/iio-oscilloscope/blob/master/dev-tools/fixup-comboboxtext.pl)

Reply:
> If Glade for GTK2 is unreliable/unusable

I mean, with docker it was not working that well. And for newer distributions it's just hard to actually find a glade 3.8 package. Even, with docker I actually had to go to ubuntu 14.04 for this...

I just used `fixup-gtk3-glade.sh`

Reply:
We're building it from source (https://ftp.gnome.org/pub/GNOME/sources/glade3/3.8/). It compiles on 18.04.4 but I haven't tried on newer versions.

Reply:
Ahh, I see. Didn't knew that. Not sure how it will work on ubuntu 20.04. Do you want me to try it? I can check it but I guess if it starts to become time consuming we can just use this glade file (since we are on the run)

Reply:
It think we should use this glade file as it is. Please run the fixup-comboboxtext.pl script on it.

Reply:
Ok. In the next time, I will try to remember this. At my office machine I run arch linux which does have glade-gtk2 build that works. But on ubuntu there's none and I tried to do it with docker and sharing the X env but that worked horribly....